aesara/graph/op.py

```diff
@@ -400,7 +401,7 @@ def R_op(
     def perform(
         self,
         node: Apply,
-        inputs: List[Variable],
+        inputs: Sequence[np.ndarray],
```

Comment:
@brandonwillard, the documentation strongly recommends to use `Sequence` instead of `List` for the function arguments. See https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance

```suggestion
        inputs: Sequence[Any],
```

Reply:
...so do you want to apply my suggestions here?
Otherwise please feel free to edit, or apply whatever you think is the right call. Just be aware that `List` might provoke a mypy warning.

Reply:
Sorry, yeah, that seems like the best choice, at least until we figure out if there's a way to narrow down the actual contents of the `Sequence` (e.g. via parameterization).  In other words, it would be _ideal_ to have that argument typed as `Sequence[np.ndarray]` when it has those values.

Reply:
@brandonwillard could you please accept and merge the suggestion that you think is the right call? This is a two line change at most and it's taking a toll to come back to such a PR over and over again.
thanks